src/api/app/models/project.rb

```diff
@@ -1750,6 +1753,13 @@ def has_local_distribution(project_name, repository)
         linked_repository.name == repository
     end
   end
+
+  def update_staging_workflow
```

Comment:
This is not really updating it right? Confusing name.

Reply:
Maybe `update_staging_workflow_on_backend`?

Reply:
sure why not :smile: